101-webapp-including-insights/azuredeploy.json

```diff
@@ -0,0 +1,130 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "webappName": {
+      "type": "string",
+      "minLength": 1,
+      "metadata": {
+        "description": "The name of the web application"
+      }
+    },
+    "appInsightsName": {
```

Comment:
move to a variable

Reply:
done

Reply:
doesn't look like it was moved to a variable